src/main.js

```diff
@@ -105,6 +105,19 @@ class Entry {
     this.defaultValue = defaultValue;
     this.comment = comment;
   }
+
+  html(){
+    return outputHTML([this]);
+  }
+}
+
+class EntryList {
+  constructor(entries) {
+    this.entries = entries;
+  }
+  html(){
+    return outputHTML(this.entries)
```

Comment:
We need `EntryList.html()` to return a Promise that will collect, concatenate and provide the output HTML code as an argument to the `resolve` argument.

To collect the output HTML from all entries `Entry.html()` methods, you can use the [`Promise.all`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) method

## Example

```javascript
const envSpecTxt = `
ADMIN_NAME
ADMIN_EMAIL: email
DATABASE_URL: url
`
const entryList = new EntryList(checkValidationOfValues(envSpecTxt))

entryList.html().then(output => {
  const outputContainer =  document.getElementById("output-container");
  outputContainer.innerHTML = output;
});
```